kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/aggregation/settings/KoverParametersProcessor.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.aggregation.settings
+
+import kotlinx.kover.gradle.aggregation.settings.dsl.intern.KoverSettingsExtensionImpl
+import org.gradle.api.provider.HasMultipleValues
+import org.gradle.api.provider.ProviderFactory
+
+
+internal object KoverParametersProcessor {
+    fun process(settingsExtension: KoverSettingsExtensionImpl, providers: ProviderFactory) {
+        if (providers.gradleProperty("kover").isPresent) {
+            settingsExtension.coverageIsEnabled.set(true)
+        }
+
+        settingsExtension.reports.includedProjects.readAppendableArgument(providers, "kover.classes.from")
```

Comment:
I guess `kover.classes.from` should reads as "collect coverage for classes from following projects", but for me options like `kover.classes.from.excludes` and `kover.classes.excludes` are not easily distinguishable.
Did you consider naming package-related options as `kover.packages.includes` and `kover.packaged.excludes`?

Reply:
The names are not final at all and this is the first thing that came to mind. 
In any case, you can come up with a lot of spelling options and they are all likely to be similar to each other in one way or another (

There will most likely be no packages - this filter is duplicated classes, and instead of `kover.classes.from`, we can try `kover.projects` and `kover.projects.excludes`